www/src/components/audits/Audits.js

```diff
@@ -44,7 +44,7 @@ function resourceInfo({
     })
   }
 
-  if (repository) return { link: `/repositories/${repository.id}`, text: `Repository{${repository.name}}` }
```

Comment:
This is a small fix since we do not have `repositories` route registered and the audit links weren't working.